avocado/core/multiplexer.py

```diff
@@ -397,11 +397,8 @@ def parse(self, args):
 
```

Comment:
Sure, it's fair to use commit message as a scratch pad.  As we move along the review process, it is also a good idea to think of other people reading the message and correlating that to the code.  It's a good idea to evolve the message.